vendor/secp256k1_support/Cargo.toml

```diff
@@ -5,10 +5,11 @@ authors = [ "CoBloX developers <team@coblox.tech>" ]
 edition = "2018"
 
 [dependencies]
-hex = "0.3"
 lazy_static = "1"
 rand = "0.3"
 secp256k1 = { version = "0.12", features = ["rand"] }
 serde = { version = "1", features = ["derive"] }
+
 [dev-dependencies]
+hex = "0.3.2"
```

Comment:
```suggestion
hex = "0.3"
```

Reply:
With dependabot, it doesn't really matter anymore I think.